modules/solid_properties/doc/content/modules/solid_properties/index.md

```diff
@@ -0,0 +1,207 @@
+# Solid Properties Module
+
+The Solid Properties module provides a consistent interface to solid properties. Unlike
+fluid properties that are often times only a function of pressure and temperature
+(or two other state variables), solid properties are frequently a function of many
+other parameters in addition to temperature, such as accumulated radiation damage,
+oxidation state, and microstructure. Some common properties include thermal
+properties such as specific heat, thermal conductivity, and emissivity; and
+mechanical properties such as elastic modulus and Poisson ratio. This module uses a
+flexible design to permit extension by other applications to include dependencies on
+application-specific parameters. In addition, a consistent interface exists to access
+derivatives with respect to the primary variables. This module allows different solids
+to be used in the input files by simply swapping the name of the Solid Properties
+Material in a plug-and-play manner.
+
+!alert warning
+Always verify that the material properties provided in this module agree with
+the system you are modeling. The selection of particular correlations here does not
+constitute an endorsement of the accuracy of those correlations.
+
+This module provides solid properties for different solid materials, organized according to
+the overall type of property.
+
+## Solid properties
```

Comment:
Link to the individual pages here instead? Same for with ThermalSolidPropetiesMaterial?

Reply:
Not sure what you mean? Replace all the documentation in this markdown file with links to the various class files?